portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.servlets;
+public class Comment {
+  public long id;
+  public String comment;
+  public long timestamp;
```

Comment:
Can these variables be `private`? In general it's a good practice to make variables `private` unless we explicitly need to make them public, protected, or package-private (where we omit the modifier entirely).

Reply:
Yes (Addressed )